### ISSUE TITLE
use secure storage

### ISSUE THREAD

**Issue by lutostag** [lutostag/easy2fa#1]
Optionally allow use of gnome keyring for storing the secrets.

**Comment by nicola-zanardi** [comment]
agreed, even just set up a passphrase at the first easy2fa startup and encrypt the yaml file
